# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../longrun/makefile-install.patch
# Copyright (C) 2004 - 2007 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2001-02-14 11:53:22.000000000 +0100
+++ ./Makefile	2003-07-12 13:50:17.000000000 +0200
@@ -1,3 +1,8 @@
+
+PREFIX = /usr
+DESTBIN = $(DESTDIR)$(PREFIX)/sbin
+DESTMAN = $(DESTDIR)$(PREFIX)/man/man1
+
 all: longrun README
 
 longrun: longrun.c
@@ -8,3 +13,14 @@
 
 clean:
 	-rm -f longrun README
+
+install:
+	install -m 755 -d $(DESTBIN)
+	install -m 755 longrun $(DESTBIN)
+	install -m 755 -d $(DESTMAN)
+	install -m 644 longrun.1 $(DESTMAN)
+
+uninstall:
+	rm -f $(DESTBIN)/longrun
+	rm -f $(DESTMAN)/longrun.1
+
